cobalt/browser/BUILD.gn

```diff
@@ -56,11 +56,11 @@ target(final_executable_type, "cobalt") {
     ":browser_switches",
     "//cobalt/base",
     "//net",
+    "//third_party/icu:icudata",
```

Comment:
Is this dep really needed here? I'm trying to understand why targets outside of `icu:icuuc` would need to depend on `icu:icudata`.